source/docs/contributing/images/example.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="Electron" modified="2020-04-04T23:22:47.383Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/12.6.5 Chrome/80.0.3987.86 Electron/8.0.0 Safari/537.36" etag="4e_k7_sMNY4Ddl0NmqiL" version="12.6.5" type="device"><diagram id="tHqYu6qWHXjSS9A0nAWg" name="Page-1">jZNLb4JAEIB/DccmPFLLVdDaJrYXk9brlh1h4z7osgj213coA0iNiSd2vnk/8KJUtRvLyuLNcJBe6PPWi1ZeGAZ+HOOnI+eexLHfg9wKTkYT2IkfGDyJ1oJDNTN0xkgnyjnMjNaQuRlj1ppmbnYwcp61ZDlcgV3G5DX9FNwV1MWjP/EXEHkxZA580ig2GBOoCsZNc4GitRel1hjXv1SbguyGN8yl93u+oR0Ls6DdPQ7JR5O+f9X71/220tvlQnwfNw8U5cRkTQ2vW6ZK7D70V4Lllimq3p2HkThoMWFSOCURBPisnDVHSI00Fok2Gi2Tg5DyH2JS5BrFDEsG5MkJrBM47CUplOC8S5M0hXCwK1nW5WzwtJBZU2sOXTc+SlQ4BoD25kSCcc54oGAUOHtGE3III1oN3Wb4RHIzbTpaECsutjzulNF15WPsaQH4oB0M4rTrP93FHxOtfwE=</diagram></mxfile>
```

Comment:
Put this in a images/source/ subdirectory.

Reply:
Yea this is still unfinished, was just trying to test something on CI, I will wait to solve the other issues, not sure if the example is actually necessary